### ISSUE TITLE
SP: Edit Team Data Table - Now Due Aug 19

### ISSUE THREAD

**Issue by staceypark** [lzim/teampsd#457]
**ORIGINATOR**
*Please follow the steps below to open an issue for the MTL Workgroups to address. Be sure to provide a descriptive title for the issue when saving.*

**Step 1** - Please assign this issue to the issue_tracker, under the “Projects” menu on the right side of your GitHub page.

**Step 2** - Please answer the questions below by placing a lowercase "x" between the applicable brackets below and providing brief narratives and screenshots where appropriate:

1. Is this issue a problem, such as a malfunction in the simulation, an error in documentation or similar concern, or is this a new idea
   we should consider?
   
   - [x] Problem 
   - [ ] New idea
   
2. What team are you on (exg., Menlo Park, Team Blue)?
na
3. How urgent is this issue?

   - [x] Urgent - *should be addressed right away!*
   - [ ] Non-urgent - *can be prioritized within normal workstreams.*

4. In the space provided below, please describe the information or behavior you expected from the product or otherwise describe what you expected to see or hear. 

**1. Change label to "User-supplied Gap Threshold" in the Team Data Table of the Sim UI as well as in the "i" information labels and body text**

**2. Remove "(Median)" as this is wrong!**

**3. This makes the Sim Team Data Table consistent with SPParams and the "i" information (which are already consistent and correct).**

**4. Add the default gap threshold for each setting to the "i" information in the SP Team Data Table in the Sim UI.**

- **39 weeks (9 months) is the default gap threshold for all 3 care settings.**

5. Please provide a screenshot of the bug, if possible.  Make as many screenshots as you like that would be helpful to illustrate the issue.

![image](https://user-images.githubusercontent.com/31089501/61234909-1c115700-a6e9-11e9-9f0d-d1cf40d8cd62.png)

12. Is there a deadline or other scheduled constraint driving your request that we should be aware of? *This will help us to prioritize this request.*

I believe we need it before the TAS training gets to Session 5, where we start working with the sim. This would currently be 8/5 as per Issue #303 

**THANK YOU!!**

---------------------------------------------------------------------------------------------------------------------------------------
**STOP HERE. INFORMATION BELOW IS TO BE FILLED OUT BY WORKGROUPS AS PART OF TRIAGE AND WORK BREAKDOWN PROCESS**
---------------------------------------------------------------------------------------------------------------------------------------

**Executive Summary**

*Estimated person-hours to complete:*

*Estimated date for completion:*

*Lead team for effort:*

*Key people (use @ assignment for people whose input will be necessary):*

**Step 1 - Complete the Interdependencies Table**

Workgroup Leads review information provided by originator and collaborate to identify interdependencies. Then they describe modifications that are needed in the appropriate column and estimate hours needed to make changes:
 
● *Workgroup Products* - This column identifies the workgroups and the products for which they are responsible. This column can be
  modified by a workgroup lead on an ad-hoc basis to add one-time products or can be modified to include new and enduring products.
  
● *Dependency?* - This column is used by the workgroup leads during collaboration to identify the existence of an interdependency.

● *Description of Change or Action Needed* - Workgroup leads briefly describe the change required.

● *Hours* - Workgroup leads estimate the person-hours required to complete the task.

Workgroup Products        | Dependency?  | Description of Change Needed               |  Hours Estimate
------------------------- | -------------|--------------------------------------------|-----------------
**Documentation**            
Brochures                 |              |                                            |
MTL See Guide             |              |                                            |
MTL Say Guide             |              |                                            |
MTL Videos                |              |                                            |
Post Test                 |              |                                            |
**Investigators**         |              |                                            |
Presentation              |              |                                            |
Quarterly Report          |              |                                            |
Annual Report             |              |                                            |
**Qualitative**           |              |                                            |
MTL Fidelity Checklists   |              |                                            |
Systems Thinking Codebook |              |                                            |
CFIR Codebooks            |              |                                            | 
**Quantitative**          |              |                                            |
Data UI                   |              |                                            |
Data Queries              |              |                                            |
SharePoint                |              |                                            |
Excel Outputs             |              |                                            |
ModelParameters           |              |                                            |
tt Reports                |              |                                            |
**Model**
CC                        |              |                                            |
MM                        |              |                                            |
PSY                       |              |                                            |
AGG                       |              |                                            |
SP                        |              |                                            |
**Sim User Interface**    |              |                                            |
Model Diagram             |       Y       |    Update labels in main diagram                             |4
"i"Information            |       Y       |  Update Experiments "i" information labels and definitions  | 2
Experiments               |        Y      |    Update slide labels                                      |2
Outputs/Charts            |      Y        |    Update chart labels in dropdowns                           |2
Exports                   |       Y       |     Update export labels                                       |  2
Master Crosswalk Table    |       Y       |  Update label changes in CW table                 | 1

**Step 2 - Identify Constraints that may affect the team’s ability to complete the necessary actions.** 

Describe the constraint in terms of capability, capacity or affect on grant research goals. If there are no constraints identified, then leave the table blank.

●  Type - The type of constraint identifies one or more resource areas that are affected.  

●  Workgroup - this is the workgroup responsible for developing a new or modifying an existing product.

●  Discussion/recommendation - A brief narrative that expands the information regarding the constraint and ideally a recommendation for
   overcoming the constraint.

**Type Constraint (check all that apply) / Workgroup(s) Affected / Discussion/Recommendation**

- [ ] Capacity      
- [ ] Cost         
- [ ] Schedule     
- [ ] Interference 
- [ ] Capability   
- [ ] Other (describe)

**Step 3 - Identify supported milestones.** *Affix the milestone at the issue level using available GitHub milestones.*

**Step 4 - Indicate the due date for completing the actions identified above, assuming constraints are removed.**

**Step 5 - Identify the lead workgroup for the action (check one):**

- [ ] Documentation
- [ ] Investigators
- [ ] Quantitative
- [ ] Qualitative
- [ ] Model
- [ ] Sim UI

**Step 6 - Go to the top of this section and complete Executive Summary information**

Note to Workgroups:  Use the issue thread to document further discussion, alternatives development and estimates.  However, be sure to update this form if dates, team responsibility or other information changes.

**Comment by staceypark** [comment]
![image](https://user-images.githubusercontent.com/31089501/61236446-dfdff580-a6ec-11e9-9dbc-ed10178f2dce.png)

1. Change "New Patient Wait Time" in the SP team data table and in the "i" information table to: **New Care Episode Wait Time** 
3. In the i information table, change the text to say : "**An estimate of the wait time..**" instead of "An estimate of new patient wait time.."

**Comment by staceypark** [comment]
@holbrooa note we decided to use the phrasing: **New Episode of Care Wait Time** instead of **New Care Episode Wait Time**

**Comment by holbrooa** [comment]
That means **New Episode of Care Start Rate** too. I just did it on the data UI end, and it's there at the top of the table on the sim UI side too.

**Comment by holbrooa** [comment]
It's in various places in the description too. @jamesmrollins will have to control+f it: we're replacing the phrase **care episode** with the phrase **episode of care** everywhere.

**Comment by staceypark** [comment]
@jamesmrollins additional context to what @holbrooa said above: 
**Patient** (like in New Patient Wait Time) was the original label, which we decided initially to change to **Care Episode** during the Manuscript Monday meeting, then decided again to say **Episode of Care** as this language is more familiar to clinicians.

**Comment by jamesmrollins** [comment]
@holbrooa @staceypark  @lzim The project just spent a lot of effort (money) in May, changing all these labels to "XXX New Care Episode Start Rate," which in my view is a more active voice.  While I would be happy to bill out more hours, I would like the PI to weigh in here first.  

With respect to all other changes, I will get an estimate.  We should check for other dependencies, especially documentation.

**Comment by saveth** [comment]
@staceypark @holbrooa @jamesmrollins @lzim It'll also require changes in the tt report as a dependency.

**Comment by lzim** [comment]
> @holbrooa @staceypark @lzim The project just spent a lot of effort (money) in May, changing all these labels to "XXX New Care Episode Start Rate," which in my view is a more active voice. While I would be happy to bill out more hours, I would like the PI to weigh in here first.

@jamesmrollins - Thanks for checking on this and considering costs. Can you provide me with an estimate of the cost of incorporating this change, so that I can make a more informed decisions?

Thanks!

**Comment by jamesmrollins** [comment]
@holbrooa will you update the ModelParameters file in OSF so I can have the latest?  I want to make sure I get the correct language.  The file in OSF appears to be an old version (6/17), if I read @staceypark correctly in the issue report, there is a new version?

**Comment by holbrooa** [comment]
@jamesmrollins 

Right. I just uploaded a new version to osf.

**Comment by lzim** [comment]
FYI All @holbrooa @saveth @TomRust @staceypark @dlkibbe @TomRust @dlounsbu @branscombj 

I approve the SP edits to the Sim UI based on the cost estimate provided by @jamesmrollins 

Thanks!

Lindsey

**Comment by jamesmrollins** [comment]
@saveth @TomRust @jessfroe if you could review this and enter your dependencies and estimated time, that would be helpful to establish the overall delivery date.  I can support having my bit done and available in TEST by July 19.

**Comment by jessfroe** [comment]
@jamesmrollins it sounds like we don't currently have all of the MTL videos for the SP module quite yet, so it may be a while before I am able to review for video edits. That being said, I will ensure that this change is on my list of items to be checking for once I am able to review the relevant videos and send any potential video fixes to Ann to determine time and cost estimate. I hope that answers your question!

**Comment by saveth** [comment]
@jamesmrollins I'm estimating 2-3hrs to update the ttreport for tracking down the variable names and changing it in the code at the various places. I'll work off the new ModelParameter, ModelCrosswalk, and testing site for the sim once you give me the okay that your part is done.

**Comment by jamesmrollins** [comment]
Thank you @saveth - I am assuming you could be done by Friday?  @jessfroe okay, got it.  But we may need to make a decision to either wait until education materials are up to date -or go ahead and promote the changes to PROD.

**Comment by jamesmrollins** [comment]
@hirenp-waferwire this is released for DEV.  Please find tasks outlined below:

1.  Change the label of SP Model as shown in the example below for all labels that include the term "Care Episode."  Find all labels in GMH, SMH and PC/PCMHI model diagrams and pop-ups.

![image](https://user-images.githubusercontent.com/30132017/61423945-977c2f80-a8c6-11e9-85c6-f31d3db0c58e.png)

2.  Change labels of Experiment Section sliders in GMH, SMH and PC/PCMHI models as shown in the examples below:
![image](https://user-images.githubusercontent.com/30132017/61424092-2721de00-a8c7-11e9-823a-6ea3e6cfb7bf.png)

![image](https://user-images.githubusercontent.com/30132017/61424198-9992be00-a8c7-11e9-9338-52314a1a5f90.png)

3.  Change labels of SP charts in Outputs and Text.

![image](https://user-images.githubusercontent.com/30132017/61424409-3bb2a600-a8c8-11e9-88a4-a5d82c289cf5.png)

4.  Update the team data table as shown below.  
![image](https://user-images.githubusercontent.com/30132017/61424546-b4b1fd80-a8c8-11e9-891e-2045abea85ac.png)

5. Update Team Data Table "i" Information as shown below:

![image](https://user-images.githubusercontent.com/30132017/61425211-0a87a500-a8cb-11e9-8021-60053b80f177.png)

6.  Ensure spreadsheet files from Home and from inside SP model export function have updated labels.

**Comment by TomRust** [comment]
@jamesmrollins @saveth @TomRust @jessfroe @lzim -- Like Savet, this will take me a couple of hours to track every instance down and then confirm with the Master Crosswalk.

@jamesmrollins -- You're updating the master crosswalk, right?  When you do, could you highlight all the updated rows?  Having that as a guide will help me verify that I've changed all the right variables in the Vensim.

**Comment by jamesmrollins** [comment]
@TomRust just to make a written record.  @lzim decided not to make changes to vensim model variable names at this point; rather, preferring to do more of a quarterly or semi-annual update.

**Comment by jamesmrollins** [comment]
@hirenp-waferwire Please see the edits below:

Team Data Table "i" information:

![image](https://user-images.githubusercontent.com/30132017/61872884-af911780-aea1-11e9-9810-18a08f56dacf.png)
 
Continuum of Care model diagram 1 of 2:
![image](https://user-images.githubusercontent.com/30132017/61873541-6d68d580-aea3-11e9-9a85-2ee0eb033146.png)

Contuum Care Model diagram 2 of 2.
![image](https://user-images.githubusercontent.com/30132017/61890679-30153f80-aec5-11e9-93be-69ba901b4f00.png)

**Comment by staceypark** [comment]
1. @jamesmrollins in all three settings (GMH, SMH, PC) there's a duplicate of New Episode of Care Start Rate.  I think "New Patient Wait Time to Start" is supposed to "New Episode of Care Wait Time" (3rd variable from top in team data table) or are these different variables?
Besides this error in Output variable dropdowns, I think the changes in language were thoroughly propagated throughout the sim.

![image](https://user-images.githubusercontent.com/31089501/62180239-27c56600-b304-11e9-9b0b-91d6f1bd914d.png)

2. @jamesmrollins @TomRust @lzim should there be expand buttons on the top right corner of these two grey boxes (missing from all three setting views)
in top left of default diagram view:
![image](https://user-images.githubusercontent.com/31089501/62180475-0add6280-b305-11e9-8ebc-9ed6abe45afb.png)
in top left of mental health care continuum view:
![image](https://user-images.githubusercontent.com/31089501/62180449-f6996580-b304-11e9-806d-1d2ede28ac25.png)

3. @TomRust @lzim  @branscombj I think these language changes in SP have a dependency in our guides where we tell the systems stories, but I can't quite decipher what it's referring to to begin with.
The New Patient Starts and Wait Times language below (bolded) - is that referring to what was previously "New Patient Wait Time" and now is "New Episode of Care Wait Time" or is it the "Patients Waiting to Start" variable I found in the Outputs. Also, with the addition of this "New Episode of Care" language, should "Patients Waiting to Start" actually be "Patients Waiting to Start New Episode of Care"???

SP – As predicted, we see a dramatic shift in patient from “high symptom” to “low symptom,” as both detection and care quality improve – the ratio of high to low symptom patients drops from 0.6:1 to 0.2:1 over two years. The number of patients with a suicide flag decreases by more than 50%, as more high symptom patients have their symptoms addressed before they can be flagged in the first place. However, the initial predicted reduction in Ending Rate is off-set by long-term increases in the Recommend Step Down Rate, thus not permanently impacting the new patient start rate. Both **new patient starts** and **wait times** in our clinic increase in the short-term, but fall below our historical average by 3 months, and continue to fall for the next year. However, because there have been no changes in Primary Care, the number of patients waiting to step down to PC hs increased substantially – from about 110 to 170, who are all undoubtedly waiting significantly longer than before.

**Comment by jamesmrollins** [comment]
@hirenp-waferwire please double check the chart listings and correct.  I have sent a master_crosswalk table via email.  Thanks, James.

![image](https://user-images.githubusercontent.com/30132017/62237448-d362ca80-b385-11e9-9d35-da978b3ce6e6.png)

**Comment by jamesmrollins** [comment]
@hirenp-waferwire thanks for your emailed response.  You are correct, there are two variables that were similarly named; "XXX New Patient Start Rate" and " XXX Patient Start Rate."  The "XXX New Patient Start Rate" was renamed to "XXX New Episode of Care Start Rate" and the "XXX Patient Start Rate" remains the same.  I also found the "XXX Patient Start Rate" for GMH and SMH was erroneously removed from the master_crosswalk_table.xlsx listing.  This has also been corrected. I will email you an updated crosswalk.  Thanks, James

**Comment by jamesmrollins** [comment]
FYI - @staceypark @TomRust @lzim @branscombj 

Hi @hirenp-waferwire.  It appears that we forgot an expand function and icon on the GMH/PC/PCMHI/SMH General Mental Health Stepped Care page (see first picture).  There is a corresponding collapse icon, but no expand icon (see second picture).

Location where expand icon should be:

![image](https://user-images.githubusercontent.com/30132017/62324590-bea63580-b45e-11e9-831e-fc618e976a62.png)

Location of corresponding collapse icon:

![image](https://user-images.githubusercontent.com/30132017/62324720-f3b28800-b45e-11e9-8f74-c42883f2e02b.png)

**Comment by staceypark** [comment]
Is this shadow variable supposed to have a collapse icon?
(shadow variable is in GMH patients waiting to start care & the rate dial is in GMH stepped care)

![image](https://user-images.githubusercontent.com/31089501/62326668-85bc8f80-b463-11e9-8d6b-5493ec19067a.png)

**Comment by jamesmrollins** [comment]
@staceypark - I think we decided it was just as easy to close the pop-up window?

**Comment by dlkibbe** [comment]
@staceypark just curious, are the numbers in the radial dials on your screen shot supposed to be rounding to 2 decimal places?

**Comment by staceypark** [comment]
@dlkibbe I'm using a rather old file so probably yes, but may be the fault of the file I'm using

**Comment by jamesmrollins** [comment]
@dlkibbe @staceypark Those numbers are gibberish (exponentials), because the Model Parameters file team data inputs are all 0.  I think I need to talk to Tom about zero proofing the model.

**Comment by staceypark** [comment]
@jamesmrollins Actually on second thought, the Patients Early in Care is just a shadow variable for the corresponding stock in the same view on the lower right hand corner. Is it still supposed to have a collapse?

**Comment by jamesmrollins** [comment]
@staceypark can you confirm you are using the "ModelParameters.xlsx" file?

**Comment by staceypark** [comment]
@jamesmrollins I'm using 100a1_abc_teama_2018_11_02.xlsx. Didn't bother changing it since I was looking at design changes not data validation.

**Comment by jamesmrollins** [comment]
@staceypark I'm surprised it even ran with a file that old.

**Comment by jamesmrollins** [comment]
@staceypark regarding the Patients Early in Care Issue.  I think that it could have both functions, meaning that it is a shadow variable referring to the source, which is the Stock towards the bottom of the diagram. It could also have functionality to take the user down to Measurement Based Stepped Care for Suicide Prevention, which is where else it has an effect. In conclusion, it probably won't hurt to add the icon.

**Comment by lzim** [comment]
> @TomRust just to make a written record. @lzim decided not to make changes to vensim model variable names at this point; rather, preferring to do more of a quarterly or semi-annual update.

@jamesmrollins is right about what I think will lead to less risk, which is periodically releases these types of fixes. I worry about these changes on a model (under the hood) level, which could stop us cold in our tracks if we don't get it perfect. 

However, @TomRust that is based on the assumption that this is cleaner and more efficient to make changes like this all at once.  Perhaps you would like to be making them in a set of models as we go that is just not yet going to be released to production. Thoughts?

**Comment by lzim** [comment]
> @dlkibbe @staceypark Those numbers are gibberish (exponentials), because the Model Parameters file team data inputs are all 0. I think I need to talk to Tom about zero proofing the model.

@staceypark @holbrooa @TomRust @jamesmrollins There's zero proofing the model, and then there's the new Data UI feature that produces zeros intentionally across several modules under some conditions. This lead to some confusion two weeks ago during the session 3 training, which folks were just getting clearer about next week. We still need more documentation to make sure we get their training right.

- We will need SEE/SAY guides that make sure folks don't upload a file to the Sim that only has 1 module (aka 2 tab) of complete data in it, but is uploaded as if it's a complete set of parameters.
- We thought that was fine, as in - the Team produces MMParams, and then works through the MM module, it's fine if the other tabs are empty...But, that's only until they want to switch to PSY and are confused because they don't remember that the file they uploaded was only one module's worth of data.
- Perhaps it's something that trains learners to make sure they have a complete set of parameters before upload.
- @holbrooa Did we talk about adding to the file naming something dynamic that reflects the module drop-down names. Like if I only select MM params, it will add MM to my file name?

FYI: @branscombj and @dlounsbu

**Comment by lzim** [comment]
Thanks @staceypark you've identified several things here, I'm not sure if they've been followed-up on or deserve checking in about them during tomorrow's workgroup leads meeting?

>@TomRust @lzim @branscombj I think these language changes in SP have a dependency in our guides where we tell the systems stories, but I can't quite decipher what it's referring to to begin with.
The New Patient Starts and Wait Times language below (bolded) - is that referring to what was previously "New Patient Wait Time" and now is "New Episode of Care Wait Time" or is it the "Patients Waiting to Start" variable I found in the Outputs. Also, with the addition of this "New Episode of Care" language, should "Patients Waiting to Start" actually be "Patients Waiting to Start New Episode of Care"???

>SP – As predicted, we see a dramatic shift in patient from “high symptom” to “low symptom,” as both detection and care quality improve – the ratio of high to low symptom patients drops from 0.6:1 to 0.2:1 over two years. The number of patients with a suicide flag decreases by more than 50%, as more high symptom patients have their symptoms addressed before they can be flagged in the first place. However, the initial predicted reduction in Ending Rate is off-set by long-term increases in the Recommend Step Down Rate, thus not permanently impacting the new patient start rate. Both new patient starts and wait times in our clinic increase in the short-term, but fall below our historical average by 3 months, and continue to fall for the next year. However, because there have been no changes in Primary Care, the number of patients waiting to step down to PC hs increased substantially – from about 110 to 170, who are all undoubtedly waiting significantly longer than before.

>@jamesmrollins FYI: any changes we make will need a change to the Help Section variable names as well, not just for SP, but the other models as well. Can you remind me - is that something I am doing or you?

**Comment by lzim** [comment]
@jessfroe on Tuesday when you're back. Could you help within on more pass on the "Episode of Care" language and whether each instance has been caught? 🙏 Please ??

**Comment by jamesmrollins** [comment]
@lzim @staceypark  I believe that Stacey has the source file (word document) for Terms and Abbreviations.  I was hoping that Stacey could upload the file in OSF under the master crosswalk project, and then I can take over keeping it up-to-date.  If Stacey can't find the file, I can recreate it and accomplish the same.  Just let me know @staceypark.

@lzim @TomRust regarding the zero-proofing issue.  I checked that out and found that it indeed is not a zero-proofing issue; rather, it is a old data parameters file issue.

**Comment by jamesmrollins** [comment]
@hirenp-waferwire please find SP changes below.  Note that these changes are across all care settings (i.e., SMH, GMH and PC/PCMHI). 

**1.** Change the label of the "GMH Patient Start Rate" dial. 

![image](https://user-images.githubusercontent.com/30132017/63136843-b84ca900-bf88-11e9-90ec-320e4b884bb4.png)


**2.** Change the lable of the "XXX New Episode of Care Start Rate" to "XXX Total New Episode of Care Start Rate."

![image](https://user-images.githubusercontent.com/30132017/63134176-2be8b900-bf7d-11e9-9019-73189c57a57a.png)

**3.** Add a collapse icon in the "XXX Patients Waiting to Start Care" pop up.

![image](https://user-images.githubusercontent.com/30132017/63132004-83365b80-bf74-11e9-8bd2-838108692b8e.png)

**4.** In the Experiments Section Team Data Table, change the label "New Episode of Care Start Rate (mean)(wks)," to "New Episode of Care Start Rate (mean)(pts/wk)." 

![image](https://user-images.githubusercontent.com/30132017/63132390-cfce6680-bf75-11e9-8f53-1c19333130c3.png)

**5** Change wording for New Episode of Care Start Rate in Team Data Table.

![image](https://user-images.githubusercontent.com/30132017/63137762-52fab700-bf8c-11e9-8925-22e686aa96dc.png)

**6** Also in the Experiments Section Team Date Table, add "(median)(wks)" central tendency and units of measure to "Time from Flag to Step up/down."

![image](https://user-images.githubusercontent.com/30132017/63133260-3dc85d00-bf79-11e9-8a59-2486ec533eaa.png)

**7.** Change "XXX High Risk Flag Rate" and "XXX High Risk Unflag Rate" as shown. Update change the chart labels in the Expanded Outputs screen and "i" Information.

![image](https://user-images.githubusercontent.com/30132017/63133449-2473e080-bf7a-11e9-94de-68dc20c76e28.png)

**8.** Change "XXX High Risk Flag Stepped Care Rate" dial label as shown - Update chart labels in the Expanded Outputs screen and "i" Information.

![image](https://user-images.githubusercontent.com/30132017/63137280-702e8600-bf8a-11e9-93cd-8a4c7cd6e5d3.png)

**9.** @jamesmrollins changed the "i" Information text for "XXX Recommend New Episode of Care Start Rate" to the following: "Set the number of XXX New Episodes of Care recommended for this setting each week. This value replaces the base case value estimated from team data. In the Base Case, team data for the XXX New Episode of Care Start Rate is used as  the inflow for the XXX Recommend New Episode of Care Rate (see the model diagram in XXX Patients Waiting to Start Care pop-out). Therefore, the Base Case "BC" on XXX Recommend New Episode of Care Rate slider corresponds to the XXX New Episode of Care Start Rate in the Team Data Table and SPParams tab."

![image](https://user-images.githubusercontent.com/30132017/63133191-e1fdd400-bf78-11e9-8d3d-bd68348fdc58.png)

**10.** @jamesmrollins changed the "i" information text for "XXX Total New Episode of Care Start Rate" to "This is the total GMH New Episode of Care rate estimated by the model keeping track of the three sources of referrals to the team (GMH, SMH, PC/PCMHI)." **This will show up when variable labels are updated.**

**11.** Change the PC/PCMHI Patient Start Rate" label and add "GMH Total New Episode of Care Start Rate" dial, label and "i" Information icon.

![image](https://user-images.githubusercontent.com/30132017/63135454-cc8da780-bf82-11e9-8c54-e80dbb1239bb.png)

12. Change "SMH Patient Start Rate" label and Change the "GMH to SMH Step Up Rate" label and variable mapping.

![image](https://user-images.githubusercontent.com/30132017/63136101-3e66f080-bf85-11e9-8bf9-e2a2c858ad9f.png)

**Comment by lzim** [comment]
@staceypark @jamesmrollins I believe that this issue can be closed is that right?

**Comment by jamesmrollins** [comment]
Hi @lzim.  Yes, the SIM UI group's work is complete.
